package.json

```diff
@@ -37,7 +36,6 @@
     "jsdom": "^19.0.0",
     "lodash": "^4.17.21",
     "pulp": "^15.0.0",
-    "purescript": "^0.14.5",
```

Comment:
Please do not remove these from here. We rely on locally installed PureScript and Spago to ensure reproducible builds.

Reply:
updated